zenoblend/node_system.py

```diff
@@ -303,6 +303,15 @@ def draw_buttons(self, context, layout):
         row.operator("node.zeno_apply", text="Apply")
         row.operator("node.zeno_stop", text="Stop")
 
+class ZenoNode_LineViewer(def_node_class('LineViewer', [('PrimitiveObject', 'prim', ''), ('bool', 'display:', '1')], [], 'blender')):
+    '''Zeno specialized LineViewer node'''
+   
+    def update(self):  # rewrite update function
+        if bpy.context.scene.zeno.executing:
+            bpy.ops.node.zeno_apply('EXEC_DEFAULT')
```

Comment:
Sounds cool, maybe this should be added to all ZenoNodes? But please only execute when the current graph is selected to be `Static` in current `Scene`.

Reply:
OK, I will check it. In fact, only static nodegroups have been tested so far. I will next test this node for framed/cached nodegroups.